server/client.go

```diff
@@ -1196,6 +1196,14 @@ func (c *client) markConnAsClosed(reason ClosedState, skipFlush bool) bool {
 	if skipFlush {
 		c.flags.set(skipFlushOnClose)
 	}
+	// Be consistent with the creation: for routes and gateways,
+	// we use Noticef on create, so use that too for delete.
+	if c.kind == ROUTER || c.kind == GATEWAY {
+		c.Noticef("%s connection closed: %v", c.typeString(), reason.String())
```

Comment:
I think if yo use %s then you do not need reason.String() and can just do reason.

Reply:
Yep.. I fixed that.